### DISCUSSION TITLE
I18n support

### DISCUSSION THREAD

**Discussion by xtexx** [lowlighter/metrics#1233] [Ideas]
Maybe I can provide some translations?

**Comment by lowlighter** [comment]
Hello 👋 !

I feel like i18n should be part of [community templates](https://github.com/lowlighter/metrics/tree/master/source/templates/community#-creating-community-templates)

The reasoning behind is that each time a new plugin or feature comes out, most of the time only the classic plugin is updated to support it because it's already a lot of work to support all templates, so supporting existing templates times X languages would be even more time consuming, or would just end up in unmaintained or outdated templates as it exponentionnaly requires energy to support

However I don't mind linking community templates in this repository though, so these i18n templates could be created outside this repository and referenced here
